drud/ddev
# Allow omit_containers["db"] for projects

## The Problem/Issue/Bug:

#1490 requested the ability to not create the db (mariadb) container in cases where it's not used.

People want this when they're using sqlite, or postgres, or another custom database setup.

## How this PR Solves The Problem:

* Allow omit_containers["db"] at the project level (not global)
* If omit_containers["db"] the dba container is also omitted because it's useless
* Remove most commands that access the db in the case it's omitted (like export_db)
* Update ddev describe to leave stuff out.

## Manual Testing Instructions:

* Use omit_containers["db"] and see how it goes
* Make sure things work without it

## Automated Testing Overview:

No changes at this point

## Related Issue Link(s):

## Release/Deployment notes:
